http/agent.go

```diff
@@ -74,6 +77,7 @@ var defaultAgentOptions = &agentOptions{
 	Timeout:         3 * time.Second,
 	MaxWaitTime:     60 * time.Second,
 	PostContentType: defaultPostContentType,
+	MaxParallel:     5,
```

Comment:
As a follow-up, can we make this configurable somehow?

Reply:
You can configure it with the config functions. I added one for MaxParallel, for example:

```golang
  a := http.NewAgent().WithMaxParallel(10)
```